### PR TITLE
Fix Conflicts between reactive and non-reactive libraries

### LINKED ISSUE #297: Conflicts between reactive and non-reactive libraries
This time it's in many-extensions module
https://github.com/quarkus-qe/quarkus-test-suite/runs/3651178337?check_suite_focus=true
I believe underlying reason to be https://github.com/quarkusio/quarkus/issues/19875

### PR DESCRIPTION
**Pull request by pjgg**
Close #297

Related to: https://github.com/quarkusio/quarkus/issues/20278

### CODE REVIEW

File: spring/spring-web/src/main/java/io/quarkus/ts/spring/web/boostrap/web/SimpleController.java [hunk 1]
```diff
@@ -19,7 +20,8 @@
     @Inject
     Template home;
 
-    @RequestMapping(value = "/", produces = MediaType.TEXT_HTML)
+    // TODO https://github.com/quarkusio/quarkus/issues/20278
+    @RequestMapping(value = "/", produces = MediaType.TEXT_HTML, method = RequestMethod.GET)
```
Comment (rsvoboda) [inline spring/spring-web/src/main/java/io/quarkus/ts/spring/web/boostrap/web/SimpleController.java:24]: present in Quarkus 2.2.3 or just in Quarkus main?

File: super-size/many-extensions/pom.xml [hunk 2]
```diff
@@ -69,10 +69,11 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-narayana-jta</artifactId>
         </dependency>
-        <dependency>
-            <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-spring-web</artifactId>
-        </dependency>
+        <!-- https://github.com/quarkusio/quarkus/issues/19190 -->
```
Comment (rsvoboda) [inline super-size/many-extensions/pom.xml:72]: How is spring-web related to `Multiple mechanisms present that use the same credential transport HttpCredentialTransport" Error` ?

**Review by rsvoboda — CHANGES_REQUESTED** [review]

File: spring/spring-web/src/main/java/io/quarkus/ts/spring/web/boostrap/web/SimpleController.java (hunk 1 above)
Comment (pjgg) [reply spring/spring-web/src/main/java/io/quarkus/ts/spring/web/boostrap/web/SimpleController.java:24]: Quarkus 2.2.3.Final (The following combination works): 
`quarkus-spring-web`
`quarkus-resteasy-qute`

Quarkus 2.2.3.Final (doesn't work): 
`quarkus-spring-web`
`quarkus-resteasy-reactive-qute`

Quarkus upstream is the opposite
Works:
`quarkus-spring-web`
`quarkus-resteasy-reactive-qute`

Doesn't works
`quarkus-spring-web`
`quarkus-resteasy-qute`

File: super-size/many-extensions/pom.xml (hunk 2 above)
Comment (pjgg) [reply super-size/many-extensions/pom.xml]: Is not really related, was a mistake. The commitment that pushes these changes is this one: 
https://github.com/quarkusio/quarkus/commit/149d5381208fa731914a1de2ab8fc868934cfdd1

**Review by rsvoboda — APPROVED** [review]

**Pull request merged**